app/helpers/organisations_helper.rb

```diff
@@ -2,7 +2,11 @@
 
 module OrganisationsHelper
   def show_checklist?(organisation, agent)
-    1.week.ago < organisation.created_at && agent.admin_in_organisation?(organisation)
+    if agent.conseiller_numerique?
+      agent.invitation_accepted_at > 2.weeks.ago && agent.admin_in_organisation?(organisation) && agent.rdvs.count < 2
+    else
+      1.week.ago < organisation.created_at && agent.admin_in_organisation?(organisation)
+    end
```

Comment:
Pinaillage de style: on peut mettre en commun le critère “admin in organisation”, et écrire les comparaisons de date dans le même sens.

```suggestion
    return false unless agent.admin_in_organisation?(organisation)

    if agent.conseiller_numerique?
      agent.invitation_accepted_at > 2.weeks.ago && agent.rdvs.count < 2
    else
      organisation.created_at > 1.week.ago
    end
```

Reply:
Et du coup, utiliser la date d’invitation comme critère pour tout le monde?

Reply:
yep, j'avais pas osé trop toucher à l'existant, mais c'est vrai que c'est plus cohérent comme ça